docs/SFTPCopy-action.md

```diff
@@ -1,51 +1,51 @@
 SFTP Copy
 =========
 
-<a href="https://cdap-users.herokuapp.com/"><img alt="Join CDAP community" src="https://cdap-users.herokuapp.com/badge.svg?t=sftp-actions"/></a>
-[![Build Status](https://travis-ci.org/hydrator/sftp-actions.svg?branch=develop)](https://travis-ci.org/hydrator/sftp-actions) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) <img src="https://cdap-users.herokuapp.com/assets/cdap-action.svg"/>
+[![Join CDAP community](https://cdap-users.herokuapp.com/badge.svg?t=sftp-actions)](https://cdap-users.herokuapp.com)
+[![Build Status](https://travis-ci.org/hydrator/sftp-actions.svg?branch=develop)](https://travis-ci.org/hydrator/sftp-actions)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![CDAP Action](cdap-users.herokuapp.com/assets/cdap-action.svg)](http://docs.cask.co/cdap)
 
-
-SFTP copy allows copying of the files from the specified directory on SFTP servers and write them to HDFS as the destination.
+SFTP Copy allows copying of the files from the specified directory on SFTP servers and write them to HDFS as the destination.
 The files that are copied can be optionally uncompressed before storing. The files are copied directly to HDFS without needing any additional staging area.
 
-
 Usage Notes
 -----------
-In order perform SFTP copy, we require host and port on which the SFTP server is running. SFTP implements secure file
-transfer over SSH. Typically port number 22 is used for SFTP(which is also default port for SSH). We also require valid
-credentials in the form of user name and password. Please make sure that you are able to SSH to the SFTP server using
-specified user and password. SSH connection to SFTP server can be customized by providing additional configurations
-such as enable host key checking by setting configuration property 'StrictHostKeyChecking' to 'yes'. These additional
-configurations can be specified using `Properties for SSH` section.
-
-Directory on the SFTP server which needs to be copied can be specified using `Source directory` property. The specified
-directory should exist and absolute path to the directory must be provided. If directory is empty then execution will
+In order to perform a SFTP copy, we require the host and port on which the SFTP server is running. SFTP implements secure file
+transfer over SSH. Typically, port number 22 is used for SFTP (which is also default port for SSH). We also require valid
+credentials in the form of a username and password. Please make sure that you are able to SSH to the SFTP server using
+the specified username and password. The SSH connection to SFTP server can be customized by providing additional configurations
```

Comment:
to the SFTP server can customized with options such as host key checking by setting...Additional configurations can be specified in the `Properties for SSH`...